cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -258,10 +251,22 @@ download_content_from_url() {
   local -r download_url=$1
   local -r output_name=$2
   local -r info_str=$3
+  local -r auth_header="Authorization: Bearer $(get_default_vm_credentials)"
+
+  info "Downloading ${info_str} from ${download_url}"
+
+  local args=(
+    -sfS
+    "${download_url}"
+    -o "${output_name}"
+  )
+  if [[ "${download_url}" == "https://storage.googleapis.com"* ]]; then
```

Comment:
Is this check necessary? This check seems a bit brittle (storage.cloud.google.com is also an acceptable hostname for downloading files from GCS). Would setting the authorization header be good enough for every HTTP request here?

Reply:
I tried to add the authorization header when request to "https://chromium.googlesource.com/chromiumos/third_party/kernel/" and got 403 error. I guess we can't just blindly add the header.

Reply:
Ok, I guess we can leave this for now, although it isn't great.